src/compute/src/render/reduce.rs

```diff
@@ -292,8 +292,9 @@ where
     // we must have more than one arrangement to collate
     soft_assert_or_log!(
         arrangements.len() > 1,
-        "Building a collation of {} arrangements, but expected more than one",
-        arrangements.len()
+        "Building a collation of {} arrangements, but expected more than one in dataflow {}",
+        arrangements.len(),
+        debug_name
```

Comment:
```suggestion
        "Building a collation of {} arrangements, but expected more than one in dataflow {debug_name}",
        arrangements.len()        
```

Reply:
This is very much a nit, but I think it's good practise to write more readable string interpolation! There are a few other places where we can do the same.